data-prepper-api/src/main/java/org/opensearch/dataprepper/model/buffer/AbstractBuffer.java

```diff
@@ -112,6 +112,11 @@ public void writeAll(Collection<T> records, int timeoutInMillis) throws Exceptio
         }
     }
 
+    @Override
+    public void writeBytes(final byte[] bytes, final String key, int timeoutInMillis) throws Exception {
+        throw new RuntimeException("not supported");
```

Comment:
There is an `UnsupportedOperationException`

Reply:
Will do this in the next PR